nrfs/.clang-format

```diff
@@ -0,0 +1,105 @@
+# SPDX-License-Identifier: GPL-2.0
```

Comment:
Please remove this file, I don't think we need it in this repo.

Reply:
Especially that the license is GPL-2.0 and thus some companies with strict no-GPL policies in embedded software may find it unappealing (however this file does not affect the license of the compiled code, but it's simpler to not have to determine copyleft scope).